toolchains/gcc-cxx11.cmake

```diff
@@ -0,0 +1,23 @@
+# Sample toolchain file for building with gcc compiler
+#
+# Typical usage:
+#    *) cmake -H. -B_build -DCMAKE_TOOLCHAIN_FILE="${PWD}/toolchains/gcc.cmake"
+
+# find compiler
+find_program(CMAKE_C_COMPILER gcc)
```

Comment:
To my knowledge, toolchain files are supposed to set only (cache) variables. CMake will ensure the path to the compiler to be valid. No need to do this manually.